### ISSUE TITLE
createProject is not adding extra compilerOptions to tsconfig.json

### ISSUE THREAD

**Issue by perezperret** [ivogabe/gulp-typescript#353]
**Expected behavior:** I want to override the `rootDir` on my `tsconfig.json` so I can transpile my tests from a `spec` directory to a `test` directory, and my project from a `src` directory to a `dist` directory, using a single config, and a single 'typings' definitions directory. If I specify the `rootDir` in two different config files (ex: `tsconfig1.json`, `tsconfig2.json`), this works fine. If I try to add the option using `ts.createProject` it doesn't. (I am running the latest `gulp-typescript`: `v2.13.4`)

**Actual behavior:** If I use `ts.createProject('tsconfig.json', {rootDir: 'spec'}` it will simply compile from both `spec` and `src` (all typescript files in the root directory) into `test` (the directory specified with `gulp.dest`), thus, I suppose `createProject` is simply ignoring the added options. Again, note that if I specify `"{ compilerOptions": { "rootDir": "spec" } }` in my `tsconfig.json`

**Your gulpfile:**

*Include your gulpfile, or only the related task (with `ts.createProject`).*
```javascript
const gulp = require('gulp');
const ts = require('gulp-typescript');

// This attempts to add the rootDir compiler option
const tsTest = ts.createProject('tsconfig.json', {rootDir: 'spec'});
gulp.task('build:spec', function() {
  return tsTest.src()
  .pipe(ts(tsTest))
  .pipe(gulp.dest('test'));
});

```

**tsconfig.json**

*Include your tsconfig, if related to this issue.*
```json
{
  "compilerOptions": {
    "module": "commonjs",
    "sourceMap": true,
    "target": "es5"
  }
}
```

**Comment by ivogabe** [comment]
I think this has been solved in #340

**Comment by perezperret** [comment]
Would this fix be available in the npm package's latest version? If so I will have a look at it as soon as I can so the issue can be closed confidently

**Comment by ivogabe** [comment]
It will be in the next release.

**Comment by ivogabe** [comment]
A new version with the fix has been released as 2.13.5.
